internal/api/fetcher/configapi.go

```diff
@@ -35,17 +42,47 @@ func (f *ConfigAPIFetcher) FetchAzureSEVSNPVersionList(ctx context.Context, atte
 
 // FetchAzureSEVSNPVersion fetches the version information from the config API.
 func (f *ConfigAPIFetcher) FetchAzureSEVSNPVersion(ctx context.Context, attestation configapi.AzureSEVSNPVersionGet) (configapi.AzureSEVSNPVersionGet, error) {
```

Comment:
Ideally, signatures would be somehow mapped within the configapi. Either as their own resources or as an extra field. But I am not sure how to do that yet. This is fine for now.